src/main/java/calculator/calculate/ParsingString.java

```diff
@@ -0,0 +1,81 @@
+package calculator.calculate;
+import calculator.global.GlobalConstant;
+import calculator.exception.Exception;
+import java.util.ArrayList;
+
+public class ParsingString {
+    private ArrayList<Character> delemeter;
+    private String check_slash = "";
+    private String check_line = "";
+    private char adding_delemeter;
+    private ArrayList<Integer> numlist = new ArrayList<Integer>();
+    private String input;
```

Comment:
```suggestion
    private char adding_delemeter;
    private List<Integer> numlist = new ArrayList<>();
    private String input;
```

객체지향 프로그래밍의 **다형성**에 대해서 알아보시면 좋을 것 같아요!